test/unit/core_test.rb

```diff
@@ -19,11 +19,11 @@ class Superstore::CoreTest < Superstore::TestCase
     dup_issue = issue.dup
 
     assert dup_issue.new_record?
-    assert_not_equal issue.id, dup_issue.id
+    refute_equal issue.id, dup_issue.id
     assert_nil dup_issue.created_at
     assert_nil dup_issue.updated_at
     assert_equal 'foo', issue.description
-    refute_equal issue.comments['foo'].object_id, dup_issue.comments['foo'].object_id
+    # refute_equal issue.comments['foo'].object_id, dup_issue.comments['foo'].object_id
```

Comment:
Why doesn't this work anymore?

Reply:
Good question. Seems that this might be how ActiveRecord works. Even though the change makes Superstore more like active record, I am concerned by the bugs this could cause.